drush/README.md

```diff
@@ -1 +0,0 @@
-This directory contains commands, configuration and site aliases for Drush. See http://packages.drush.org/ for a directory of Drush commands installable via Composer.
```

Comment:
I don't even know why this is part of this ticket.. I didn't do it! I cannot even find the thing.